chapel.html.markdown

```diff
@@ -310,6 +311,18 @@ stringSet += "a"; // Redundant add "a"
 stringSet -= "c"; // Remove "c"
 writeln( stringSet ); 
 
+// Both ranges and domains can be sliced to produce a range or domain with the
+// intersection of indices
+var rangeA = 1.. ; // range from 1 to infinity
+var rangeB =  ..5; // range from negative infinity to 5
```

Comment:
This is nothing wrong with your pull request, it is just more my curiosity. "negative infinity" I wonder where it *actually* starts? It can't be negative infinity, can it?